app/assets/javascripts/carts.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Rails generated files are super cool- but if you have a bunch you don't use, best practice would be to delete the empty files (though soon you will have full javascript files too! :) )